specification/appendix-a-included-utilities.md

```diff
@@ -192,23 +192,23 @@ The strategy to use is passed in to the Multi-Provider constructor as follows:
 
 ```typescript
 new MultiProvider(
-	[
-		{
-			provider: new ProviderA()
-		},
-		{
-			provider: new ProviderB()
-		}
-	],
-	new FirstMatchStrategy()
+ [
+  {
+   provider: new ProviderA()
+  },
+  {
+   provider: new ProviderB()
+  }
+ ],
+ new FirstMatchStrategy()
 )
 ```
 
 By default, the Multi-Provider uses the “FirstMatchStrategy”.
 
 Here are some standard strategies that come with the Multi-Provider:
 
-**First Match**
+###### First Match
```

Comment:
`MD036/no-emphasis-as-heading/no-emphasis-as-header Emphasis used instead of a heading`